packages/core/src/App/Constants/routes-config.js

```diff
@@ -92,7 +92,7 @@ const initRoutesConfig = () => ([
             { path: routes.cashier_acc_transfer, component: AccountTransfer,      title: localize('Transfer between accounts'), icon_component: 'IconAccountTransfer' },
             { path: routes.cashier_pa_transfer,  component: PaymentAgentTransfer, title: localize('Transfer to client'),        icon_component: 'IconAccountTransfer' },
             // To work with P2P please uncomment this line
-            // { path: routes.cashier_p2p,          component: P2PCashier,           title: localize('P2P Cashier'),            icon_component: 'IconP2PCashier' },
+            // { path: routes.cashier_p2p,          component: P2PCashier,           title: localize('OTC payment'),               icon_component: 'IconP2PCashier' },
```

Comment:
Shouldn't title be title case?

Reply:
@mahdipourziaei-binary No, none of the tabs in cashier are title case